installer/resources/lambda_submit/function.py

```diff
@@ -525,8 +525,8 @@ class AzureDataShipperCloudWatchEventTarget(CloudWatchEventTargetResource):
     event_bus_name = CloudWatchEventBusazure.get_output_attr('arn')
     target_id = 'AzureDataShipperTarget'  # Unique identifier
     target_input = json.dumps({
-        'jobName': "data-shipper-azure",
-        'jobUuid': "data-shipper-azure",
+        'jobName': "data-shipper-common",
```

Comment:
let's have it just as data-shipper, since we have a single one then no need in `common` in name.

Reply:
@mkinchin-paladin addressed this (data-shipper) instead of (data-shipper-common) in latest push